server/controllers/notification.js

```diff
@@ -0,0 +1,109 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+
+const notifications = {
+  newRequest: {
+    title: "New Request",
+    message: "You recieved a new request from ",
+  },
+  message: {
+    title: "New Message",
+    message: "You received a new message from ",
+  },
+  requestResponse: {
+    title: "Response Received",
+    message: "You received a response from ",
+  },
+};
+
+// @route POST /notification/:id
+// @desc create notification
+// @access Private
+exports.createNotification = asyncHandler(async (req, res) => {
+  const { sender, type } = req.query;
+  const userId = req.params.id;
```

Comment:
We don't need to send the user's id with path params, this information can be obtained from the request object - `req.user.id` 😄

Reply:
Fixed.